src/main/java/be/syntra/devshop/DevshopFront/services/utils/ProductMapperUtil.java

```diff
@@ -2,15 +2,24 @@
 
 import be.syntra.devshop.DevshopFront.models.Product;
 import be.syntra.devshop.DevshopFront.models.dtos.ProductDto;
+import be.syntra.devshop.DevshopFront.models.dtos.ProductDtoList;
+import be.syntra.devshop.DevshopFront.models.dtos.ProductListDto;
+import be.syntra.devshop.DevshopFront.services.CartService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
 @Component
 public class ProductMapperUtil {
```

Comment:
would you be willing to rename both these Util (Category, Product) to simply Mapper as well?

Reply:
renamed both